File: 프로그래머스/2/12941. 최솟값 만들기/최솟값 만들기.py
def solution(A,B):
    A.sort()
    B.sort(reverse=True)
    ans = 0
    for i in range(len(A)):
        ans += A[i] *B[i]
    # [실행] 버튼을 누르면 출력 값을 볼 수 있습니다.

    return ans

File: 프로그래머스/2/12941. 최솟값 만들기/최솟값 만들기.java
import java.util.*;
class Solution
{   
    public int solution(int []A, int []B)
    {
        Arrays.sort(A);
        Arrays.sort(B);
        System.out.println(Arrays.toString(A));
        int[] newB = new int[B.length];
        for(int i=B.length-1;i>-1;i--){
            newB[B.length-1-i] = B[i];
        }
        int sum =0;
        for(int i=0;i<A.length;i++){
            sum += A[i]*newB[i];
        }


        return sum;
    }
}